assets/js/components/wp-dashboard/WPDashboardIdeaHub.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Idea Hub notice component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { useFeature } from '../../hooks/useFeature';
+import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import { MODULES_IDEA_HUB } from '../../modules/idea-hub/datastore/constants';
+import GoogleLogoIcon from '../../../svg/logo-g.svg';
+import Link from '../Link';
+
+const { useSelect } = Data;
+
+function WPDashboardIdeaHub() {
+	const isFeatureEnabled = useFeature( 'ideaHubModule' );
```

Comment:
Let's rename this variable to something different not to collide with the utility function that has the same name.
```suggestion
	const isIdeaHubEnabled = useFeature( 'ideaHubModule' );
```